test/serverTest.js

```diff
@@ -1,14 +1,18 @@
-const sleep = (millis) => new Promise(resolve => setTimeout(resolve, millis))
+// eslint-disable-next-line func-style
+const sleep = millis => new Promise(resolve => setTimeout(resolve, millis))
```

Comment:
I'm OK with this. But was it just weird looking to have it a normal `function` declaration? It also doesn't need to be at the very top of the file if that's the part that would make it look funny.

Reply:
I don't think it would look particularly weird, no, I just figured it might have been done like this for a reason since there are other things that are actually written out functions going on lower down 😅 I just changed it to this and put it at the top of function declarations and all the tests still pass, so I can update it if that looks nicer!

```
function sleep(millis) {
  return new Promise(resolve => {
    setTimeout(resolve, millis)
  })
}
```

Reply:
Top of the function declarations seems good to me. Thanks!
It was probably just put there like that because it was copied as a one-liner from a Stackoverflow answer :smile: